features/lwipstack/lwipopts.h

```diff
@@ -93,8 +93,22 @@
 #define LWIP_RAW                    0
 #endif
 
+#ifdef MBED_CONF_LWIP_MEMP_NUM_TCPIP_MSG_INPKT
+#define MEMP_NUM_TCPIP_MSG_INPKT    MBED_CONF_LWIP_MEMP_NUM_TCPIP_MSG_INPKT
+#endif
+
+#ifdef MBED_CONF_LWIP_TCPIP_MBOX_SIZE
+#define TCPIP_MBOX_SIZE             MBED_CONF_LWIP_TCPIP_MBOX_SIZE
+#else
 #define TCPIP_MBOX_SIZE             8
+#endif
+
+#ifdef MBED_CONF_LWIP_DEFAULT_TCP_RECVMBOX_SIZE
```

Comment:
This doesn't need an `#ifdef`. Default 8 can be in the JSON.

Reply:
done